pkg/kubelet/kubelet.go

```diff
@@ -2025,11 +1992,11 @@ func (kl *Kubelet) cleanupOrphanedPodDirs(pods []*api.Pod, runningPods []*kubeco
 	}
 	errlist := []error{}
 	for _, uid := range found {
-		if active.Has(string(uid)) {
+		if allPods.Has(string(uid)) {
 			continue
 		}
-		if volumes, err := kl.getPodVolumes(uid); err != nil || len(volumes) != 0 {
-			glog.V(3).Infof("Orphaned pod %q found, but volumes are not cleaned up; err: %v, volumes: %v ", uid, err, volumes)
+		if volumeDirsExist, err := kl.podVolumesExist(uid); err != nil && volumeDirsExist {
```

Comment:
Comment here explaining that under these conditions the volume may actually still be in use.